org.guitarix.Guitarix.appdata.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
```

Comment:
Doesn't it need `<launchable>`? Also, should be upstreamed.

Reply:
Fixed.